src/windows/main_window.py

```diff
@@ -1974,6 +2062,9 @@ def setup_toolbars(self):
         spacer.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
         self.videoToolbar.addWidget(spacer)
 
+        # Other controls (right-justified)
```

Comment:
It's a terribly minor nit, but I'd change this to "right-aligned" instead of "right-justified", because to "justify" in the printing sense means "to line up evenly" — in other words, "right-justified" text isn't necessarily text that's slid all the way over to the right, but rather it's text that forms an even vertical right margin, as opposed to normal left-justified text which has a ragged right margin. You can't really "justify" a single horizontal line of icons, or anything.

Of course, you _could_ claim that the **exact** same constraints apply to the word "aligned", and I'd be forced to admit that you've got me there, so feel free to just ignore me and I'll shut up.

Reply:
I have no particular argument either way, so I'm good with making the change. 😜